common-data/amplitude-sync-choices.json

```diff
@@ -1,4 +1,5 @@
 [
-  ["USERS", "Users"],
+  ["USERS", "Users v1 (deprecated)"],
```

Comment:
Note that the presence of the word "deprecated" makes the `export_to_amplitude` command ignore the event type.